### DISCUSSION TITLE
Adding the vixen/babyv

### DISCUSSION THREAD

**Discussion by mattc2x4** [manna-harbour/miryoku#296] [Q&A] [answered]
is there any documentation or processes to add compatibility for new keyboards? looking to add the [vixen.](https://eyeohdesigns.com/products/vixen)  This is a great project and I already have 2 keyboards running miryoku.

**Comment by manna-harbour** [comment, marked as answer]
The keyboard definition isn't in QMK master but is available at https://github.com/joedinkle/vial-qmk/tree/eod-keebs/keyboards/eyeohdesigns/vixen.

First you'll need to merge that with Miryoku QMK. Ordinarily that could be done with the workflow automatic merge function, but at the moment it will need to be copied manually. See https://github.com/manna-harbour/miryoku/discussions/287. 

The keyboard definition is also in a vial fork, so automatic merge probably would have failed anyway.

Other than that you need to add a mapping to Miryoku QMK. See https://github.com/manna-harbour/miryoku/discussions/264#discussioncomment-7689140. Refer to https://github.com/joedinkle/vial-qmk/blob/2c320bab1548f1684e8957dc7775a6fe720b5d91/keyboards/eyeohdesigns/vixen/keymaps/default/keymap.c#L23.

If you don't have 3 x 1u thumb keys you'll also need to add an alternative layout option to change the thumb key positions and enable thumb combos.

Mappings can also be added by request.

**Reply by mattc2x4** [reply]
thanks for your response. if you'd be willing to add the mapping @manna-harbour that'd be great. I'm pretty short on time in my personal life right now. I am solely interested in a 36 key map, but as time becomes more available I'd love to add a 34 key version myself.

**Reply by mattc2x4** [reply]
if there's a more formal way to request a mapping, I'd be happy to do it.
